server/gc/src/test/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogsTest.java

```diff
@@ -234,4 +257,288 @@ public void testIsUUID() {
     assertFalse(GarbageCollectWriteAheadLogs.isUUID("0" + UUID.randomUUID().toString()));
     assertFalse(GarbageCollectWriteAheadLogs.isUUID(null));
   }
+
+  @Test
+  public void testTimeToDeleteTrue() throws InterruptedException {
+    gcwal.clearFirstSeenDead();
+    HostAndPort address = HostAndPort.fromString("tserver1:9998");
+    long wait = AccumuloConfiguration.getTimeInMillis("1s");
+    gcwal.timeToDelete(address, wait); // to store first seen dead
+    sleep(wait * 2);
+    assertTrue(gcwal.timeToDelete(address, wait));
+  }
+
+  @Test
+  public void testTimeToDeleteFalse() {
+    HostAndPort address = HostAndPort.fromString("tserver1:9998");
+    long wait = AccumuloConfiguration.getTimeInMillis("1h");
+    long t1, t2;
+    boolean ttd;
+    do {
+      t1 = System.nanoTime();
+      gcwal.clearFirstSeenDead();
+      gcwal.timeToDelete(address, wait);
```

Comment:
should this first call to timeToDelete always return false?  If so could wrap it w/ `assertFalse()`